listen.js

```diff
@@ -0,0 +1,5 @@
+const app = require("./app");
+
+app.listen(9090, () => {
+  consoel.log(`listening on port ${9090}`);
```

Comment:
This doesn't need to be a template literal if you're passing in a literal value. You could've just written the whole thing as a static string.